website/docs/intro/install/deb-step2.sh

```diff
@@ -1,3 +1,4 @@
 sudo install -m 0755 -d /etc/apt/keyrings
-curl -fsSL https://packages.opentofu.org/opentofu/tofu/gpgkey | sudo gpg --no-tty --batch --dearmor -o /etc/apt/keyrings/opentofu.gpg
+curl -fsSL https://get.opentofu.org/opentofu.gpg | sudo tee /etc/apt/keyrings/opentofu.gpg >/dev/null
+curl -fsSL https://packages.opentofu.org/opentofu/tofu/gpgkey | sudo gpg --no-tty --batch --dearmor -o /etc/apt/keyrings/opentofu-repo.gpg
```

Comment:
This is needed because the repository is signed with a different key than the packages. However, since our GPG key is added, we always have the option to move to our own package repo.